src/OpenTK.NT/Native/User32/Cursor.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Security;
+
+using HCURSOR = System.IntPtr;
+using HINSTANCE = System.IntPtr;
+
+namespace OpenTK.NT.Native
+{
+    public static partial class User32
+    {
+        public static class Cursor
+        {
+            public static HCURSOR LoadCursor(CursorName lpCursorName)
+                => LoadCursor(IntPtr.Zero, new IntPtr((int)lpCursorName));
+
+            [DllImport("user32.dll", SetLastError = true)]
+            public static extern HCURSOR LoadCursor(
+                [In] [Optional] HINSTANCE hInstance,
+                [In] [MarshalAs(UnmanagedType.LPTStr)] string lpCursorName
+            );
+
+            [DllImport("user32.dll", SetLastError = true)]
+            public static extern HCURSOR LoadCursor(
+                [In] [Optional] HINSTANCE hInstance,
+                [In] IntPtr lpCursorName
```

Comment:
Is this parameter defined as a string or an untyped pointer?

Reply:
Either a pointer to a string, or a pre-defined integer value (see the `CursorName` enum).

This is an oversight, I'll change it to string and int instead of string and IntPtr.